onlineweb4/settings/base.py

```diff
@@ -144,16 +144,6 @@
     'guardian.backends.ObjectPermissionBackend',
 )
 
-#Django 1.9 passwordvalidation
-AUTH_PASSWORD_VALIDATORS = [
-    {
-        'NAME': 'django.contrib.auth.password_validation.MinimumLengthValidator',
-        'OPTIONS': {
-            'min_length': 6,
-        }
-    },
-]
-
```

Comment:
Didn't mean to do that, I reverted the entire base file to the develop version when I noticed my changes from working on the 500-page was still on mye computer, and I forgot I'd changed anything. I'll put it back in when I get back on the right computer ;)